src/.nuget/Microsoft.NETCore.Runtime.CoreCLR/Microsoft.NETCore.Runtime.CoreCLR.pkgproj

```diff
@@ -17,6 +17,9 @@
     <Dependency Include="Microsoft.NETCore.Windows.ApiSets">
       <Version>1.0.1-rc3-23915</Version>
     </Dependency>
+    <Dependency Include="Microsoft.NETCore.Jit">
```

Comment:
This whole method of static dependencies sucks.  We need something that is maintainable and plugs into whatever build coordination that's going on.  /cc @markwilkie